### PR TITLE
[ethercalc] add guide for ethercalc

### LINKED ISSUE #594: Guide for ethercalc

### PR DESCRIPTION
**Pull request by pxlfrk**
Hi, i've added a guide for [Ethercalc](https://ethercalc.net/).

I've tested the guide on a fresh uberspace and it's running, but the ethercalc-instance won't connect to the redis server according to the supervisord-logs, see also Support-Ticket #473934.

Maybe you have an idea how to fix the issue :)

closes #594

### CODE REVIEW

**Comment by pxlfrk** [comment]
With the help of @luto i fixed the issue & added the missing lines to the guide.

File: source/guide_ethercalc.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  * :manual:`domains <web-domains>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`web-backends <web-backends>`
+  * :lab:`redis <guide_redis>`
+  
+
+License
+=======
+
+Redis is distributed under different licenses, including `Common Public Attribution License (Socialtext Inc.)`_, `Apache License 2.0 (SheetJS)`_ and `MIT License`_.
```
Comment (luto) [inline source/guide_ethercalc.rst:38]: why "redis"?

File: source/guide_ethercalc.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+============
+
+Install node module
+-------------------
+
+Install the node module using the provided node packet manager `npm`: 
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ npm install ethercalc
```
Comment (luto) [inline source/guide_ethercalc.rst:68]: This drops `node_modules` directory into the users home. Please put it into a directory to keep things clean.
```suggestion
 [isabell@stardust ~]$ mkdir ethercalc
 [isabell@stardust ~]$ cd ethercalc
 [isabell@stardust ethercalc]$ npm install ethercalc
```

File: source/guide_ethercalc.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+.. note:: Ethercalc tries to connect to Redis via TCP/IP, which will accepts connections on both TCP/IP and UNIX socket. If you configured redis by following the :lab:`guide <guide_redis>` it is currently configured *for UNIX socket only* as this is also the more stylish, secure and faster version. So by now the connection between Ethercalc and Redis won't be established.
+Adding the environment variable ``REDIS_SOCKPATH`` pointing to our the unix-socket to the ``supervisord``-config forces Ethercalc to connect to Redis through the UNIX socket.
+
+
+.. code-block:: ini
+
+ [program:ethercalc]
+ command=%(ENV_HOME)s/node_modules/ethercalc/bin/ethercalc
+ environment=NODE_ENV="production",REDIS_SOCKPATH ="%(ENV_HOME)s/.redis/sock"
```
Comment (luto) [inline source/guide_ethercalc.rst:112]:
```suggestion
 environment=NODE_ENV="production",REDIS_SOCKPATH="%(ENV_HOME)s/.redis/sock"
```

File: source/guide_ethercalc.rst (hunk 1 above)
Comment (pxlfrk) [reply source/guide_ethercalc.rst:38]: Typo, fixed.

File: source/guide_ethercalc.rst (hunk 2 above)
Comment (pxlfrk) [reply source/guide_ethercalc.rst:68]: fixed in the guide.

File: source/guide_ethercalc.rst (hunk 3 above)
Comment (pxlfrk) [reply source/guide_ethercalc.rst:112]: remove blankspace.

**Comment by luto** [comment]
Thanks for your guide :)

We're a bit swamped with customer support currently, so it might take us a bit to review it. While we're busy, you can make sure that you're using [the snippets](https://github.com/Uberspace/lab/tree/master/source/includes) for domain setup, supervisord, mysql and other tasks. Sorry for the delay!

**Comment by luto** [comment]
Thank you! :tada:

**Pull request merged**